dev/cnf/gradle/scripts/fat.gradle

```diff
@@ -324,6 +331,59 @@ assemble {
   }
 }
 
+ext.gitClone = { proj, branch = 'master', folder = '', site = 'git@github.com', org = 'eclipse' ->
+
+  // We need to know which MP project at least
+  if( proj == null || proj.length() == 0 ){
+    project.logger.lifecycle("The first parameter to task clone is the project name and cannot be defaulted");
+    return
+  }
+
+  // Default the destination forlder to the project name
+  if( folder == null || folder.length() == 0 ){
+    folder = proj
+  }
+
+  //Staging area parent folder to enable copying all subfolders.
+  File gitReposDir = new File(project.buildDir, 'gitRepos')
+
+  // If running locally use git directly
+  if( System.properties['fat.test.localrun'] ){
+
+    File repoDir = new File(gitReposDir, folder)
+    String into = repoDir.getAbsolutePath()
+     
+    String url = " " + site + ":" + org + "/" + proj + ".git "
+    String cmd = "git clone --single-branch --branch " + branch + " " + url + " " + into
```

Comment:
Could possibly add `--depth 1` to this list of arguments to avoid downloading all the history of the repository.

Reply:
Done.